Documentation/Signoffs/AcceleratorSystem.md

```diff
@@ -0,0 +1,51 @@
+Accelerator Pedal Position System
+--------------------------------------
+### Function of the Subsystem
+The accelerator pedal position system will take the OEM signal from the pedal by splicing into the signal wires and input them to the local microcontroller. The local microcontroller will then calculate the position based the change of the voltage from the OEM hall effect sensor.
+
+### Specifications & Constraints
+- Must use OEM accelerator pedal 
+- Must not have any exposed or loose wires according to NESC standard
+- Must send voltage in the range that changes with pedal position 
+- Must have wires able to handle 6 V and 1 A 
+- Shall have an accurate measure if the position
+- Shall send position signal to local MCU
+
+### Wiring Schematic
```

Comment:
You have many connections here to OEM wiring. How? You can't leave components out of the schematic